### PR TITLE
Return active version of a  location

### LINKED ISSUE #267: Update getLocation service method to return the latest version of a location

### PR DESCRIPTION
**Pull request by Rkareko**
Resolves #267

### CODE REVIEW

File: src/test/java/org/opensrp/repository/postgres/LocationRepositoryTest.java [hunk 1]
```diff
@@ -254,7 +255,7 @@ public void testUpdateLocation() throws ParseException {
 		
 		assertNotNull(updatedLocation);
 		assertEquals(GeometryType.POLYGON, updatedLocation.getGeometry().getType());
-		assertEquals(PropertyStatus.PENDING_REVIEW, updatedLocation.getProperties().getStatus());
+		assertEquals(PropertyStatus.ACTIVE, updatedLocation.getProperties().getStatus());
```
Comment (githengi) [inline src/test/java/org/opensrp/repository/postgres/LocationRepositoryTest.java:258]: Why is this modified

Comment (Rkareko) [reply src/test/java/org/opensrp/repository/postgres/LocationRepositoryTest.java:258]: The `get({identifier})` method has been updated to return the `active` location.
This section of the test has also been updated https://github.com/OpenSRP/opensrp-server-core/pull/268/files#diff-185cc4dc43c21f1f9757f284ef3621afR245

File: src/test/java/org/opensrp/repository/postgres/LocationRepositoryTest.java [hunk 2]
```diff
@@ -250,7 +254,7 @@ public void testUpdateLocation() throws ParseException {
 		physicalLocation.getProperties().setEffectiveEndDate(effectiveEndDate);
 		physicalLocation.setJurisdiction(true);
 		locationRepository.update(physicalLocation);
-		PhysicalLocation updatedLocation = locationRepository.get("3734");
+		PhysicalLocation updatedLocation = locationRepository.get("3734", true, 0);
```
Comment (githengi) [inline src/test/java/org/opensrp/repository/postgres/LocationRepositoryTest.java:257]: you can assert ` locationRepository.get("3734");` return null
